src/Fixer/DescribedFixerInterface.php

```diff
@@ -10,13 +10,20 @@
  * with this source code in the file LICENSE.
  */
 
-namespace PhpCsFixer\ConfigurationException;
+namespace PhpCsFixer\Fixer;
+
+use PhpCsFixer\FixerDefinitionInterface;
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
- *
- * @internal
+ * @author SpacePossum
  */
-final class UnallowedFixerConfigurationException extends InvalidFixerConfigurationException
+interface DescribedFixerInterface
```

Comment:
I think we'll see a lot of custom fixers with dummy docs as we cannot force people to set something useful there.